cmd/grype/cli/commands/db_import.go

```diff
@@ -27,14 +31,43 @@ func DBImport(app clio.Application) *cobra.Command {
 }
 
 func runDBImport(opts options.Database, dbArchivePath string) error {
-	dbCurator, err := distribution.NewCurator(opts.ToCuratorConfig())
+	// TODO: tui update? better logging?
+
+	if isLegacy(dbArchivePath) {
+		return legacyDBImport(opts, dbArchivePath)
+	}
+	return importDB(opts, dbArchivePath)
+}
+
+func importDB(opts options.Database, dbArchivePath string) error {
+	client, err := distribution.NewClient(opts.ToClientConfig())
+	if err != nil {
+		return fmt.Errorf("unable to create distribution client: %w", err)
+	}
+	c, err := installation.NewCurator(opts.ToCuratorConfig(), client)
+	if err != nil {
+		return fmt.Errorf("unable to create curator: %w", err)
+	}
+
+	if err := c.Import(dbArchivePath); err != nil {
+		return fmt.Errorf("unable to import vulnerability database: %w", err)
+	}
+	return stderrPrintLnf("Vulnerability database imported")
+}
+
+func legacyDBImport(opts options.Database, dbArchivePath string) error {
+	dbCurator, err := legacyDistribution.NewCurator(opts.ToLegacyCuratorConfig())
 	if err != nil {
 		return err
 	}
 
 	if err := dbCurator.ImportFrom(dbArchivePath); err != nil {
-		return fmt.Errorf("unable to import vulnerability database: %+v", err)
+		return fmt.Errorf("unable to import vulnerability database: %w", err)
 	}
 
 	return stderrPrintLnf("Vulnerability database imported")
 }
+
+func isLegacy(path string) bool {
+	return !strings.Contains(filepath.Base(path), "vulnerability-db_v6")
```

Comment:
Since a rename might make this return true when it shouldn't, it might be worth inverting to `isNew` for now. It might be nice to have a config or a less brittle check or something at some point.